lib/app.js

```diff
@@ -42,12 +43,8 @@ class AppView extends React.Component {
   constructor(props) {
     super(props);
 
-    // Metrics
-    Metrics.launch();
-    Metrics.install();
-
+    track.appLaunch();
```

Comment:
@isabelrios here is where we're tracking 'app-launch'. Inside this function I also track 'first-app-launch'. I renamed this from 'install' to make it clearer what was actually being tracked; although you could assume that this is a similar time that the install completed.